recordset/index.html.in

```diff
@@ -10,5 +10,8 @@
         <div class="container-fluid">
         %TEMPLATES%
         </div>
+        <div ng-controller="recordsetController">
+            <footer ng-show="vm.hasLoaded"></footer>
```

Comment:
I don't think there is a reason to not always show the footer. You don't need to wait for the app page to load.